File: CPKINGDOM/CPKINGDOM.Core/Models/BulkItems.cs
using System.Collections.Generic;

namespace CPKINGDOM.Core.Models
{
    public class BulkItems
    {
        public int SupplierId { get; set; }
        public string DateReceived { get; set; }
        public List<Inventory> SelectedItems { get; set; }
    }
}

File: CPKINGDOM/CPKINGDOM.Portal/ClientApp/src/app/models/bulk-items.ts
import { Inventory } from "./inventory";

export class BulkItemModel {
    supplierId: number;
    dateReceived: string;
    selectedItems: Inventory[];
    constructor() {
        this.supplierId = 0;
        this.dateReceived = '';
        this.selectedItems = [];
    }
}